domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -77,6 +79,36 @@ object InteractionObjectTestBuilder {
     ).build()
   }
 
+  /** Creates fraction part for [NumberWithUnits].
+   */
```

Comment:
This is incorrect.
Suggested change:
`/** Creates fraction part for [NumberWithUnits]. */`

Reply:
Done